android_configure.py

```diff
@@ -65,6 +65,9 @@ def patch_android():
 os.environ['PATH'] += os.pathsep + toolchain_path + "/bin"
 os.environ['CC'] = toolchain_path + "/bin/" + TOOLCHAIN_PREFIX + android_sdk_version + "-" +  "clang"
 os.environ['CXX'] = toolchain_path + "/bin/" + TOOLCHAIN_PREFIX + android_sdk_version + "-" + "clang++"
+# nodejs-mobile patch: add host CC and CXX
+os.environ['CC_host'] = os.popen('command -v clang').read().strip()
+os.environ['CXX_host'] = os.popen('command -v clang++').read().strip()
```

Comment:
I wonder, what's the point of this patch? following the [github workflow](https://github.com/Acurast/nodejs-mobile/blob/update-v24.5.0/.github/workflows/build-mobile.yml#L43), this will no longer use the installed `gcc`, won't it?

Reply:
It might be an oversight on my side, I went through a couple of iterations where I was trying different build systems and configurations, looking for a combination that would require the least adjustments. I remember that at one point I had issues with gcc and then switched to clang. I'll update the workflow accordingly.